src/main/java/net/neoforged/bus/api/IEventBus.java

```diff
@@ -202,5 +202,17 @@ public interface IEventBus {
      */
     <T extends Event> T post(T event);
 
+    /**
+     * Submit the event for dispatch to listeners registered with a specific {@link EventPriority}.
+     * <p>
+     * If the bus is not started yet, an exception will be thrown.
+     * <p>
+     * Note that posting events phase-by-phase is overall a lot slower than a regular {@link #post(Event)} call.
```

Comment:
Potential rephrase and addition:
```suggestion
     * Manually posting events phase-by-phase through this method is less performant than dispatching to all phases through a {@link #post(Event)} call. Prefer that method when per-phase dispatching is not needed.
```